proposals/etcd.adoc

```diff
@@ -0,0 +1,59 @@
+== etcd Project Proposal
+
+*Name of Project*: etcd
+
+*Description*:
+
+etcd is a consistent distributed key-value store, designed to hold small amounts of data that can fit entirely in memory (although etcd still writes to disk for durabilities) and mainly used as a separate coordination service for other distributed systems like Kubernetes. Typical etcd cluster is distributed over 3 to 5 nodes, for high availability, while it prioritizes consistency and partition tolerance. Which means, it provides on logical cluster view of many physical servers. So long as majority of cluster is up, etcd continues to work, even under machine failures. This redundancy provides fault tolerance.
+
+etcd server implements Raft consensus algorithm for data replication. Raft is a leader-based protocol. Data is replicated from leader to follower; follower forwards proposals to leader, and leader decides what to commit or not. Leader persists and replicates an entry, once it has been agreed by the quorum of cluster. The underlying storage layer for Raft log is write-ahead log (WAL). Committed entries are written out to disk, so they can be replayed on restart. etcd uses gRPC for transport layer. Client employs HTTP/2 Ping for server health checking, and implements automatic failover under faulty networks.
+
+*Sponsor / Advisor from TOC*: Brian Grant <briangrant@google.com>
+
+*Unique Identifier*: etcd
+
+*License*: https://github.com/coreos/etcd/blob/master/LICENSE[Apache License 2.0]
+
+*Maturity Level*: Incubating
+
+*Source control repositories*: https://github.com/etcd-io (soon to be moved here, from https://github.com/coreos/etcd/issues/9965[August 27, 2018])
+
+*Issue tracker*: https://github.com/coreos/etcd/issues
+
+*Initial committers*: https://github.com/philips[Brandon Phillips] and https://github.com/xiang90[Xiang Li] at CoreOS (see https://github.com/coreos/etcd/blob/master/MAINTAINERS)
+
+*Infrastructure requirements (CI / CNCF Cluster)*: N/A
+
+*External dependencies*: https://github.com/coreos/etcd/blob/master/bill-of-materials.json
+
+*Statement on alignment with CNCF mission*: etcd has enabled adoption of cloud native systems: etcd is "container packaged" by publishing container image for every release, and etcd is a critical component and most reliable storage implementation to many "dynamically managed" systems like Kubernetes.
```

Comment:
nit: This paragraph could do with some word-smithing.  "has enabled adoption" and "most reliable" are strong claims, and unnecessary, IMO, to justify alignment with the CNCF's mission:

https://www.cncf.io/about/charter/